ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_TOPOLOGY.yml

```diff
@@ -10,43 +10,73 @@ underlay_ospf_authentication:
       cleartext_key: please_vault_me
 
 type: l3leaf
-mgmt_interface: null # To get the coverage for get(self.cv_topology_config, "mgmt_interface")
 use_cv_topology: true
 cv_topology:
   - hostname: CV-MLAG-OSPF-L3LEAF1A
     platform: vEOS-LAB
     interfaces:
-      - name: Management1
+      - name: Management66
+      - name: Management66/1 # Ignored since we only use the first managemetn interface.
+      - name: Ethernet1
+        neighbor: DC1-SPINE4
+        neighbor_interface: Ethernet20
+      - name: Ethernet5
+        neighbor: CV-MLAG-OSPF-L3LEAF1B
+        neighbor_interface: Ethernet5
+      - name: Ethernet6
+        neighbor: CV-MLAG-OSPF-L3LEAF1B
+        neighbor_interface: Ethernet6
+  - hostname: CV-MLAG-OSPF-L3LEAF1B
+    platform: vEOS-LAB
+    interfaces:
+      - name: Management67
+      - name: Ethernet1
+        neighbor: DC1-SPINE4
+        neighbor_interface: Ethernet21
       - name: Ethernet5
-        neighbor: MLAG-OSPF-L3LEAF1B
+        neighbor: CV-MLAG-OSPF-L3LEAF1A
         neighbor_interface: Ethernet5
       - name: Ethernet6
-        neighbor: MLAG_SAME_SUBNET_L3LEAF1B
+        neighbor: CV-MLAG-OSPF-L3LEAF1A
         neighbor_interface: Ethernet6
   - hostname: CV-TOPOLOGY-LEAF3A
     platform: vEOS-LAB
     interfaces:
-      - name: Ethernet16
+      - name: Ethernet1
         neighbor: DC1-SPINE4
         neighbor_interface: Ethernet16
+      - name: Ethernet5
+        neighbor: CV-TOPOLOGY-LEAF3B
+        neighbor_interface: Ethernet5
+  - hostname: CV-TOPOLOGY-LEAF3B # Same level, not MLAG so this is ignored.
+    platform: vEOS-LAB
+    interfaces:
+      - name: Ethernet1
+        neighbor: DC1-SPINE4
+        neighbor_interface: Ethernet17
+      - name: Ethernet5
+        neighbor: CV-TOPOLOGY-LEAF3A
+        neighbor_interface: Ethernet5
 
-# Providing select default interfaces to be used by `use_cv_topology`
-# for automatically deriving the topology from the given `cv_topology`
```

Comment:
would probably keep speed for uplink and mlag to make sure it still works here

Reply:
it is not really what we are testing here and there is no code interaction, which will be kept in check by coverage.